python_modules/airline-demo/build.sh

```diff
@@ -17,19 +19,19 @@ rm -rf dagstermill/dist && \
 rm -rf .tox dist && \
 \
 docker build -t airline-demo-airflow . && \
-docker tag airline-demo-airflow dagster/airline-demo-airflow && \
-\
+docker tag airline-demo-airflow dagster/airline-demo-airflow
+
 # pip uninstall -y dagster-airflow && \
-pip install -e ../dagster-airflow && \
-\
-dagster-airflow scaffold airline_demo_download_pipeline --image airline-demo-airflow \
--e environments/local_base.yml -e environments/local_fast_download.yml --install && \
-\
-dagster-airflow scaffold airline_demo_ingest_pipeline --image airline-demo-airflow \
--e environments/local_base.yml -e environments/local_ingest.yml --install && \
-\
-dagster-airflow scaffold airline_demo_warehouse_pipeline --image airline-demo-airflow \
--e environments/local_base.yml -e environments/local_warehouse.yml --install
+# pip install -e ../dagster-airflow && \
```

Comment:
can you also check in the airflow scaffolds? also tests should ensure that we are using latest dagster-airflow scaffolds